src/Action.hs

```diff
@@ -468,12 +501,37 @@ markIdeaInResultPhase iid rv = do
 
 -- * Topic handling
 
-topicInRefinementTimedOut :: (ActionPersist m, ActionSendMail m) => AUID Topic -> m ()
+topicInRefinementTimedOut :: (ActionPhaseChange m) => AUID Topic -> m ()
 topicInRefinementTimedOut = topicTimeout RefinementPhaseTimeOut
 
-topicInVotingTimedOut :: (ActionPersist m, ActionSendMail m) => AUID Topic -> m ()
+topicInVotingTimedOut :: (ActionPhaseChange m) => AUID Topic -> m ()
 topicInVotingTimedOut = topicTimeout VotingPhaseTimeOut
 
+topicInVotingResetToJury
+    :: (ActionPhaseChange m) => AUID Topic -> m ()
+topicInVotingResetToJury tid = do
+    topic <- mquery $ findTopic tid
+    case topic ^. topicPhase of
+        PhaseVoting _ -> topicPhaseChange topic VotingPhaseSetbackToJuryPhase
+        _             -> pure ()
+
+-- * Admin activities
+
+-- | Make a topic timeout if the timeout is applicable.
+-- FIXME: Only admin can do that
```

Comment:
> Are there any objections to doing them at all?

I am totally *for* it. That is a shotgun change, and we should do that after the first release.